tools/cmake/CheckPythonPackage.cmake

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# CMake functions for checking for Python packages
+# Requires PYTHON_EXECUTABLE to be defined.  Call FindPythonInterp first!
```

Comment:
Where should `FindPythonInterp` be called?

Reply:
Oh that's just my old comment back from when I first wrote this module.  FindPython3 used to be FindPythhonInterp.